pkgs/applications/version-management/git-and-tools/tig/default.nix

```diff
@@ -33,6 +33,9 @@ stdenv.mkDerivation rec {
     make install-doc
     mkdir -p $out/etc/bash_completion.d/
     cp contrib/tig-completion.bash $out/etc/bash_completion.d/
+    mkdir -p $out/share/zsh/site-functions
+    cp contrib/tig-completion.zsh $out/share/zsh/site-functions/
```

Comment:
Reading the comment in the file it should be:

```
install -D  contrib/tig-completion.zsh $out/share/zsh/site-functions/_tig
```

mkdir is also not needed in this case.